docker-compose-fullbuild.yml

```diff
@@ -11,6 +11,8 @@ services:
       - "8000:8000"
     volumes:
       - ./frontend/src/:/opt/frontend/src/
+      - ./frontend/.eslintrc.js:/opt/frontend/.eslintrc.js
```

Comment:
We don't need to include this for fullbuild. Fullbuild already copies all files over during build